packages/dask-task-models-library/src/dask_task_models_library/container_tasks/errors.py

```diff
@@ -1,20 +1,20 @@
 """ Dask task exceptions
 
 """
-from pydantic.errors import PydanticErrorMixin
+from models_library.errors_classes import OsparcErrorMixin
 
 
-class TaskValueError(PydanticErrorMixin, ValueError):
-    code = "task.value_error"
+class TaskValueError(OsparcErrorMixin, ValueError):
+    code = "task.value_error"  # type: ignore[assignment]
```

Comment:
THOUGHT: I am not 100% sure are using these codes right now. Since now the new `OsparcErrorMixin` generates them by default, perhaps we can drop them?   It applies to all the custom that were previously inherited from `pydanticErrormixin`